pom.xml

```diff
@@ -43,6 +43,8 @@
         <module>csv-exporter</module>
         <module>csv-importer</module>
         <module>itools-packager</module>
+        <module>loadflow</module>
+        <module>sensitivity</module>
         <module>topology</module>
```

Comment:
I have removed the modules corresponding to new tutorials otherwise maven fails during the checks